lib/scrabble.rb

```diff
@@ -1,4 +1,92 @@
 module Scrabble
+
+LETTER_VALUES = [
+  [1, "A", "E", "I", "O", "U", "L", "N", "R", "S", "T"],
+  [2, "D", "G"],
+  [3, "B", "C", "M", "P"],
+  [4, "F", "H", "V", "W", "Y"],
+  [5, "K"],
+  [8, "J", "X"],
+  [10, "Q", "Z"]
+]
+
+class Scrabble
   # require your gems and classes here
   # require_relative 'scrabble/whatevs'
+  require "awesome_print"
+
+def self.letter_score(letter)
+  LETTER_VALUES.each do |letter_value_array|
+      if letter_value_array.include?(letter.upcase)
+        value = letter_value_array[0]
```

Comment:
Setting this to the `value` variable would make sense if you were going to reuse `value`, but i think you could just use `return letter_value_array[0]` here